docs/java/workflows.md

```diff
@@ -311,6 +311,39 @@ public class GreetingWorkflowImpl implements GreetingWorkflow {
 }
 ```
 
+### ParentClosePolicy
+
+When creating a child Workflow, you can define a `ParentClosePolicy` that terminates, cancels, or abandons the Workflow if the child's parent stops execution.
```

Comment:
The rest of this appears to be a copy & past from the top except with a Java example.
Same suggestions for this as well.